configs/example.ini

```diff
@@ -117,3 +117,9 @@ contact_string = You can contact me over at <a href="https://KKonaTestAdmin.se/c
 [pnsl]
 ; bearer token can be generated here https://bot.tetyys.com/swagger/index.html
 ;token = abcef
+
+[spotify] # register an app at https://developer.spotify.com/
```

Comment:
we've used ; for comments before, should probably stick to that. and see how we've positioned the comments as well above